README.md

```diff
@@ -5,6 +5,15 @@ See the [developer guide](https://jenkins.io/doc/developer/plugin-development/)
 
 ## Changelog
 
+### 3.0 (2018-11-30)
+
+* [JENKINS-20679](https://issues.jenkins-ci.org/browse/JENKINS-20679) - 
+Inject `Minimum-Java-Version` into the manifest.
+  * It is set by a new mandatory `minimumJavaVersion` parameter in `hpi:hpi`, `hpi:jar` and `hpi:hpl`
+  * Format: `java.specification.version` according to [Java JEP-223](https://openjdk.java.net/jeps/223). 
+    Examples: `1.6`, `1.7`, `1.8`, `6`, `7`, `8`, `9`, `11`, ...
```

Comment:
Huh? That JEP seems to say the format should be `6`, `7`, etc., and _not_ `1.6`, `1.7`, etc. as before. So which is the version format expected by this feature?

Reply:
https://openjdk.java.net/jeps/223 . `1.6` and `1.7` are the old formats of Java specification, we should not maintain them going forward imho

Reply:
Right, so why would we support them at all in this newly added feature?